docker-compose.production.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+  badgehub-frontend:
+    image: ghcr.io/badgeteam/badgehub-frontend:${FRONTEND_IMAGE_TAG:-latest}
+    ports:
+      - "8001:3000"
```

Comment:
How is the setup now? Does this compose file get used by the production setup? Or how is this supposed to be used?